### ISSUE TITLE
[Welcome] Add status box to welcome message re: currently selected network from metamask

### ISSUE THREAD

**Issue by pelsasser** [MARKETProtocol/dApp#148]
Depends on #147 being completed first

## Before you `start work`

Please read our contribution [guidelines](https://docs.marketprotocol.io/#contributing) and if there is a bounty involved  please also see [here](https://docs.marketprotocol.io/#gitcoin-and-bounties)

If you have ongoing work from other bounties with us where funding has not been released, please do not pick up a new issue.  We would like to involve as many contributors as possible and parallelize the work flow as much as possible.

Please make sure to comment in the issue here immediately after starting work so we know your plans for implementation and a timeline.

Please also note that in order for work to be accepted, all code must be accompanied by test cases as well.
### User Story

As a user I would like the welcome message to have a status of what network I am currently on, and a visual confirmation if I am on rinkeby, vs a non supported network.

### Why Is this Needed?

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: To ensure people dont get confused, and to help them see what network they are on 

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
No visual information on the welcome screen about the current network

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Visual confirmation Rinkeby is selected, or not

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)
Visit the main page of the dApp in the `develop` branch

### Solution
[comment]: # (Provide a summary of the solution and a task list on what needs to be fixed.)
*Summary*:
Add a network status box to the lower left corner of the welcome message that shows the currently selected network.  If Rinkeby, the status is green and a check mark shows up, if not Rinkeby a red warning appears there instead (or something like this)

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

- [ ] Welcome message reports back on network selected
- [ ] Follows all style guidelines 
- [ ] Tests created for all new functionality

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

__Work has been started__.

 1. [@brentoneill](https://gitcoin.co/profile/brentoneill) 

 has committed to working on this project to be completed 6 days, 23 hours from now.


@pelsasser, __please see the below comments / questions regarding approach for this ticket from the bounty hunter(s):__ 
- [@brentoneill](https://gitcoin.co/profile/brentoneill): I will use web3 to detect the network -  web3.version.network
And show a green (success) message if connected to Rinkeby and a red (warning) message that the user is not connected to rinkeby

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. **Submitted** 4. Done 

<hr>

__Work for 0.06 ETH (40.5 USD @ $674.98/ETH) has been submitted by__:

 1. [@brentoneill](https://gitcoin.co/profile/brentoneill) 



@pelsasser please take a look at the submitted work:
* [PR](https://github.com/MARKETProtocol/dApp/pull/159) by @brentoneill

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/MARKETProtocol/dApp/148/458)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
$8,126.61 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__The funding of 0.06 ETH (40.84 USD @ $680.6/ETH) attached to this issue has been approved & issued to @brentoneill.__  

 * Learn more at [on the Gitcoin Issue Details page](https://gitcoin.co/issue/MARKETProtocol/dApp/148/458)
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $8,187.89 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by darkdarkdragon** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.08 ETH (55.37 USD @ $692.1/ETH) attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](http://localhost:8000/issue/MARKETProtocol/dApp/148/693).
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $0 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
